hs_collection_resource/views.py

```diff
@@ -40,65 +40,103 @@ def update_collection(request, shortkey, *args, **kwargs):
             if collection_res_obj.resource_type.lower() != "collectionresource":
                 raise Exception("Resource {0} is not a collection resource.".format(shortkey))
 
-            # get res_id list from POST
+            # get 'resource_id_list' list from POST
```

Comment:
@pkdash 
current "test_collection.py" has several test cases for this view function. will add more to include new "update_type": "add" and "remove"